### PR TITLE
accessiblity improvements

### PR DESCRIPTION
**Pull request by russbiggs**
In response to issue #19.  Updated some page elements to improve overall accessibility. Most notably  and title attributes on other elements. Some other issues still persist such as non-unique element ids and color contrast.
These changes update the lighthouse report to 70-80 range for all pages:
https://googlechrome.github.io/lighthouse/viewer/?gist=d8b110b27f2045c7081f640c1c330f47

### CODE REVIEW

File: how-to-give-back/index.md [hunk 1]
```diff
@@ -5,4 +5,5 @@ order: 3
 color: blue
 description: How can you or your organization contribute to OSM?
 image: mapathon_MM.jpg
+image-alt: a group of people sitting at a mapping event
```
Comment (mikelmaron) [inline how-to-give-back/index.md]: @russbiggs maybe "a group of people editing OpenStreetMap on computers at a mapathon"?

File: what-is-openstreetmap/index.md [hunk 2]
```diff
@@ -5,4 +5,5 @@ order: 0
 color: blue
 description: A brief intro to OpenStreetMap
 image: GOPR7567.jpg
+image-alt: three men talking on a road
```
Comment (mikelmaron) [inline what-is-openstreetmap/index.md]: @russbiggs maybe "three men planning a field survey to collect OpenStreetMap data"

File: who-is-openstreetmap/index.md [hunk 3]
```diff
@@ -5,4 +5,5 @@ order: 1
 color: blue
 description: The people and communities of OSM
 image: whatishistory_14665124541_52ce01db2c_o_KLL.jpg
+image-alt: a group of people sitting on stairs
```
Comment (mikelmaron) [inline who-is-openstreetmap/index.md]: @russbiggs maybe "a group of people posing on stairs after completing a mapping workshop"

File: why-openstreetmap/index.md [hunk 4]
```diff
@@ -5,4 +5,5 @@ order: 2
 color: blue
 description: What makes OpenStreetMap special?
 image: whyshouldweuseOSM_niccolo_rigacci.jpg
+image-alt: three people sitting on stairs looking a notebooks
```
Comment (mikelmaron) [inline why-openstreetmap/index.md]: @russbiggs maybe "three people sitting on stairs comparing data collected during an OpenStreetMap field survey"

File: working-with-osm-data/index.md [hunk 5]
```diff
@@ -5,4 +5,5 @@ order: 13
 color: blue
 description: How to use, research and edit OSM
 image: whatisOSM_Civic_Use_DanielXOneilr.jpg
+image-alt: a person working on a laptop computer with a map on the screen
```
Comment (mikelmaron) [inline working-with-osm-data/index.md]: @russbiggs maybe " a person working on a laptop computer, editing OpenStreetMap data"

**Comment by mikelmaron** [comment]
@russbiggs thank you for these improvements. I've made recommendations for alt text which is a little more descriptive and particular to OSM. Do you like these changes? Can you add them to the PR?

**Comment by russbiggs** [comment]
@mikelmaron I tried to make reasonable alt labels but I figured you might have suggestions for more descriptive titles.  All look good to me.  I'll update the PR.

**Comment by russbiggs** [comment]
@mikelmaron I've updated all the descriptions based on your recommendations.  Let me know if you need any other changes before merging.

**Comment by mikelmaron** [comment]
thank you @russbiggs

**Pull request merged**